apps/channels/urls.py

```diff
@@ -7,5 +7,5 @@
 urlpatterns = [
     path("telegram/<uuid:channel_external_id>", views.new_telegram_message, name="new_telegram_message"),
     path("whatsapp/incoming_message", views.new_whatsapp_message, name="new_whatsapp_message"),
-    path("facebook/incoming_message", views.new_facebook_message, name="new_facebook_message"),
+    path("facebook/<slug:team>/incoming_message", views.new_facebook_message, name="new_facebook_message"),
```

Comment:
You could call this `team_slug` to stay consistent with other places. Also the teams middleware expects that name.

Reply:
https://github.com/dimagi/open-chat-studio/pull/115/commits/ce2298bedab43b56ec9d9aeb6e4fb6a1a217534e